src/main/java/org/breedinginsight/brapps/importer/model/imports/experimentObservation/ExperimentObservation.java

```diff
@@ -0,0 +1,275 @@
+    @ImportFieldMetadata(id="envLocation", name="Environment Location", description = "Location of the environment")
+    private String envLocation;
+
+    //TODO make this ImportFieldTypeEnum.INTEGER
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="envYear", name="Environment Year", description = "Year corresponding to the environment")
+    private String envYear;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="expUnitId", name="Experiment Unit ID", description = "Human-readable alphanumeric identifier for experimental units unique within environment. Examples, like plot number, are often a numeric sequence.")
+    private String expUnitId;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.INTEGER)
+    @ImportFieldMetadata(id="expReplicateNo", name="Experiment Replicate Number", description = "Sequential number of experimental replications")
+    private String expReplicateNo;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.INTEGER)
+    @ImportFieldMetadata(id="expBlockNo", name="Experiment Block Number", description = "Sequential number of blocks in an experimental design")
+    private String expBlockNo;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="row", name="Row", description = "Horizontal (y-axis) position in 2D Cartesian space.")
+    private String row;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="column", name="Column", description = "Vertical (x-axis) position in 2D Cartesian space.")
+    private String column;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="treatmentFactors", name="Exp Treatment Factor Name", description = "treatement factors in an experiment with applied variables, like fertilizer or water regimens.")
+    private String treatmentFactors;
+
+    @ImportFieldType(type= ImportFieldTypeEnum.TEXT)
+    @ImportFieldMetadata(id="ObsUnitID", name="Observation Unit ID", description = "A database generated unique identifier for experimental observation units")
+    private String ObsUnitID;
+
+    public BrAPITrial constructBrAPITrial(Program program, boolean commit, String referenceSource, UUID id) {
+        BrAPIProgram brapiProgram = program.getBrapiProgram();
+        BrAPITrial trial = new BrAPITrial();
+        if( commit ){
+            trial.setTrialName( Utilities.appendProgramKey(getExpTitle(), program.getKey() ));
+
+            // Set external reference
+            trial.setExternalReferences(getBrAPIExternalReferences(program, referenceSource, id, null, null));
+        }
+        else{
+            trial.setTrialName( getExpTitle() );
+        }
+        trial.setTrialDescription(getExpDescription());
+        trial.setActive(true);
+        trial.setProgramDbId(brapiProgram.getProgramDbId());
+        trial.setProgramName(brapiProgram.getProgramName());
+
+        // TODO: Get to a constant
```

Comment:
Could you clarify what this TODO means and if should be implemented in this card or no?

Reply:
Thanks.  No longer relevant. Will remove.